examples/serviceaccount.tf

```diff
@@ -9,4 +9,8 @@ module "serviceaccount" {
 
   namespace           = "my-namespace"
   github_repositories = ["my-repo"]
+
+  # Enabling "enable_sa_env_secret" will create serviceaccount secrets as "Environment secrets"
+  # https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-secrets
+  enable_sa_env_secret   = true
```

Comment:
Can we comment the flag here.This will avoid users who use the example which will enable the flag by default.